README.md

```diff
@@ -1,6 +1,6 @@
 <br>
 <div align="center">
-  <img alt="Aerie" height="50" src="docs/sphinx_scylladb_theme/static/img/logos/aerie-wordmark-light.svg">
+  <img alt="Aerie" height="50" src="docs/sphinx_scylladb_theme/static/img/logos/aerie-wordmark-with-background.svg">
```

Comment:
It seems to be rendering a little smaller than what is on develop. Not sure why. Can we make it the same size?

Reply:
Ah that's because I widen the SVG but not the img tag.

Reply:
Increased the img height to 60.

Reply:
It still feels a little too small for the page. Maybe increase to like 80-100?

Reply:
Increased the height to 85.